bigframes/_config/compute_options.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Options for displaying objects."""
+
+import dataclasses
+from typing import Optional
+
+
+@dataclasses.dataclass
+class ComputeOptions:
```

Comment:
Could you add an "autoclass" for this file.

Need to update the following files for this to actually get documented:

* https://github.com/googleapis/python-bigquery-dataframes/blob/main/docs/reference/bigframes/options.rst
* https://github.com/googleapis/python-bigquery-dataframes/blob/main/docs/templates/toc.yml

Reply:
Done